java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -34,6 +34,25 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> get(String key);
 
+    /**
+     * Gets a string value associated with the given <code>key</code> and deletes the key. {@see
+     * #get}.
```

Comment:
I think we can remove the pointer to GET, same applies for the Javadocs in BaseTransaction
```suggestion
     * Gets a string value associated with the given <code>key</code> and deletes the key.
```